wrench/src/yaml_frame_reader.rs

```diff
@@ -220,6 +220,28 @@ impl YamlFrameReader {
         self.builder().push_rect(rect, Some(local_clip), color);
     }
 
+    fn handle_line(&mut self, item: &Yaml, local_clip: LocalClip) {
+        let color = item["color"].as_colorf().unwrap_or(*BLACK_COLOR);
+        let baseline = item["baseline"].as_f32().expect("line must have baseline");
+        let start = item["start"].as_f32().expect("line must have start");
+        let end = item["end"].as_f32().expect("line must have end");
+        let width = item["width"].as_f32().expect("line must have width");
+        let orientation = match item["orientation"].as_str().expect("line must have orientation") {
```

Comment:
did you consider using `define_string_enum` for implementing those enum-string conversions?